aries-site/src/pages/components/menu.js

```diff
@@ -22,22 +28,147 @@ const Menu = () => {
         <Subsection name={title} level={1} topic={topic}>
           <SubsectionText>{page.description}</SubsectionText>
           <Example
-            code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/controls/MenuExample.js"
-            details={[
-              <Text size="large">
-                Menu is used to filter or sort content on a page. It is, in some
-                ways, similar to the{' '}
-                <Anchor href="input#select">Select component</Anchor>. However,
-                use the select component when the user must specify from a list
-                of options and submit.
-              </Text>,
-            ]}
+            code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/menu/MenuExample.js"
+            designer="https://designer.grommet.io/menu?id=HPE-design-system-hpedesignsystem-hpe-com"
             docs="https://v2.grommet.io/menu?theme=hpe#props"
+            figma="https://www.figma.com/file/DyFWlzLBUtK5KH5jJhwoCB/HPE-Menu-Component"
           >
             <MenuExample />
           </Example>
         </Subsection>
       </ContentSection>
+      <ContentSection>
+        <Subsection name="Guidance">
+          <SubsectionText>
+            A Menu is a set of actions that may be used to engage with content
+            on a page or to navigate to other pages. When a menu item is
+            clicked, the action will be executed. Menu is distinct from{' '}
+            <Anchor href={nameToPath('Select')}>Select</Anchor>, which is often
+            used in the context of a Form to choose from a list of values but
+            does not perform actions.
+          </SubsectionText>
+        </Subsection>
+        <Subsection name="Usage" level={3} gap="small">
+          <SubsectionText>
+            A Menu consists of a main button and a set of menu items. These
+            items are displayed when the menu button is clicked. The menu closes
+            when an item in the menu is selected or anywhere else on the page is
+            clicked. To create a predictable experience for users, the following
+            are some guidelines to follow when using Menu.
```

Comment:
Trying to streamline/focus:
```suggestion
           To create a predictable experience for users, the following
            are guidelines for using Menu.
```

If the ` A Menu consists of a main button and a set of menu items. These
            items are displayed when the menu button is clicked. The menu closes
            when an item in the menu is selected or anywhere else on the page is
            clicked. ` is needed, I'd move it to an "About" section or incorporate into the first guidance paragraphs.

Reply:
To be honest, I don't think it's needed.